raidnearby.py

```diff
@@ -658,6 +680,8 @@ def processRaidImage(self, raidfilename):
 
                 pokemon_id = database.get_raid_pokemon_id(session, gym)
                 end_time = database.get_raid_time(session, gym)
+                if end_time == 0:
```

Comment:
Need to be
`if end_time < unix_time:`
end_time will be previous raid end time for most people with RDRM.